thirdeye/thirdeye-frontend/app/pods/manage/alerts/index/route.js

```diff
@@ -68,10 +68,10 @@ export default Route.extend({
     * @method error
     * @example: from Model hook `return RSVP.reject("FAIL");` will invoke the error action
     */
-    error(error, transition) {
-      if (error) {
-        return this.transitionTo('error');
-      }
-    }
+    // error(error, transition) {
+    //   if (error) {
+    //     return this.transitionTo('error');
+    //   }
+    // }
```

Comment:
you may want to remove this route change

Reply:
Ok i can remove for now. I want to add it back once I update the error handling more.

Reply:
done.